.typos.toml

```diff
@@ -0,0 +1,2 @@
+[files]
+extend-exclude = ["crates/iceberg/testdata"]
```

Comment:
Is it possible to sth like `**/testdata`? I think all `testdata` should be skipped.